src/common/AllExperiments/AllExperiments.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0-windows</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ExperimentationLive Condition="'$(ExperimentationLive)' == ''">True</ExperimentationLive>
```

Comment:
Shouldn't this be false by default? In order to build properly in users machines?

Reply:
This whole file is still a mixture of spaces and tabs. Let's please be consistent, as Dustin previously noted.